wear/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,21 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.HomeAssistant"
         android:fullBackupContent="@xml/backup_rules">
+        <!-- Start things like SensorWorker on device boot -->
+        <receiver android:name=".sensors.SensorReceiver"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
+                <action android:name="android.intent.action.MY_PACKAGE_REPLACED" />
+                <action android:name="android.intent.action.MY_PACKAGE_SUSPENDED" />
+                <action android:name="android.intent.action.MY_PACKAGE_UNSUSPENDED" />
+                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+                <action android:name="com.htc.intent.action.QUICKBOOT_POWERON" />
+                <action android:name="android.app.action.NEXT_ALARM_CLOCK_CHANGED" />
+                <action android:name="android.bluetooth.device.action.ACL_CONNECTED" />
+                <action android:name="android.bluetooth.device.action.ACL_DISCONNECTED" />
```

Comment:
We should probably register for these once we add the corresponding sensors

Reply:
Good catch. Shall I remove it for now?

Reply:
Yes please!

Reply:
Done!